src/main/scala/io/gatling/build/automated/GatlingAutomatedScalafixPlugin.scala

```diff
@@ -25,16 +25,24 @@ import sbt._
 import sbt.Keys._
 
 object GatlingAutomatedScalafixPlugin extends AutoPlugin {
+
   override def requires: Plugins = ScalafixPlugin && GatlingBuildConfigPlugin
 
+  override def globalSettings: Seq[Def.Setting[_]] = Seq(
+    scalafixOnCompile := true
```

Comment:
Where does this setting come from?
We should not redefine a default global setting of a setting created by another plugin.

Reply:
That's what we do in GatlingAutomatedScalafmtPlugin